.gitignore

```diff
@@ -174,5 +174,6 @@ BenchmarkDotNet.Artifacts
 artifacts
 .assets
 
-# Temporary cadl folders for cadl generation
-TempCadlFiles/
\ No newline at end of file
+# Temporary folders for cadl version SDK generation
+TempCadlFiles/
+TempSDKFiles/
```

Comment:
Do we really need another temp directory? Can it go under an existing directory like artifacts?

Reply:
@m-nash What do you think?

Reply:
At least to me, it would be more appropriate to put under each SDK folders since they are input not output. Do you have any concern about this?

Reply:
@weshaggard for the prototype here lets ignore "how we get the code" and once we have everything else hooked up we will sync with you on the best way to get this information.